homework-1/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Natalia's Base Page</title>
+  <link href="css/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Baloo+Thambi+2:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="menu">
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About Me</a></li>
+        <li class="active"><a href="#">Base Page</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="place-box">
+      <h2>Ski Resort of Tyrol</h2>
+      <figure>
+        <img src="img/gletscher-stubaital.jpg" alt="Stubai Glacier">
+        <figcaption>Stubai Glacier</figcaption>
+      </figure>
+      <p><a href="https://g.page/stubai-glacier?share" target="_blank">Location</a></p>
+      <p class="place-box__description">That's not a Munich place, but I miss snowy mountains of Tyrol sooo much. I was planning to snowboard on Easter, but... Unfortunately, I can't do it again till the next season.</p>
+    </section>
+    <section class="place-box">
+      <h2>Flaucherstandl Kiosk</h2>
+      <figure>
+        <img src="img/flaucher.jpg" alt="Flaucherstandl Kiosk">
```

Comment:
I think you can put figurecaption here as well.

Reply:
thanks! line added below